src/frontmatter.ts

```diff
@@ -48,8 +48,14 @@ export const extractFrontMatter = (
         const rawMeta = open + meta + close;
         const strippedContent = content.slice(rawMeta.length);
 
+        const loadedData = load(escapeLiquid(meta), options) as FrontMatter;
+
+        if (!loadedData) {
+            return [{}, content, rawMeta];
+        }
```

Comment:
you return `content` here, but it's `strippedContent` in return below, it's OK?

Reply:
I suppose yes because it's an empty error state: https://github.com/diplodoc-platform/liquid/pull/8/files#diff-940b8b622059f703bee1bc315425448737d4619f4d399cdd0c540fb054b113a1R66